index.html

```diff
@@ -15,8 +15,8 @@ <h1 id="logo">Aziz Mejri</h1>
         <i class="fas fa-bars"></i>
       </button>
       <nav id="mobile-nav">
```

Comment:
- [ ] Please 🙏 adjust your styling to match the Figma design as required in the project requirements, The menu list should be aligned to the left, and the font color should be changed to the color provided in the Figma design . This will improve the appearance🎉 of your menu.
## Your design
![image](https://user-images.githubusercontent.com/63445083/218241155-d9dcd3b2-7803-457d-b4d4-01358519c603.png)
## Figma design
![image](https://user-images.githubusercontent.com/63445083/218241373-e1bf9a3d-57fb-487a-8e95-059a02ef7188.png)